### PR TITLE
Implement coala in uikit

### PR DESCRIPTION
**Pull request by rhemon**
This Pull Request is being done as part of the Google Code-In task https://github.com/coala/coala/wiki/Google-Code-In-Task-Implement-coala .

[See the changes](https://github.com/rhemon/uikit/commits/coala-ci)

The mentors are : @jayvdb , @sils

In addition to normal review comments, collaborators are welcome to come and discuss the task at https://gitter.im/coala/coala.

### CODE REVIEW

File: docs/overlay.html [hunk 1]
```diff
@@ -376,8 +376,7 @@ <h3 class="tm-article-subtitle">Example</h3>
 
                                 <li class="uk-width-medium-1-3">
                                     <figure class="uk-overlay uk-overlay-hover">
-                                        <img src="images/placeholder_600x400.svg" width="600" height="400" alt="Image">
-                                        <figcaption class="uk-overlay-panel uk-overlay-background  uk-overlay-bottom uk-overlay-slide-bottom">
+                                        <img src="images/placeholder_600x400.svg" width="600" height="400" alt="Image"><figcaption class="uk-overlay-panel uk-overlay-background uk-overlay-bottom uk-overlay-slide-bottom">
```
Comment (sils) [inline docs/overlay.html]: why moving those two lines into one?

**Comment by sils** [comment]
the rest lgtm after careful review, good job @rhemon ! I which someone did that to my code :P

File: docs/overlay.html (hunk 1 above)
Comment (rhemon) [inline docs/overlay.html]: was receiving Trailing Whitespace somewhere around there, so I think I brought figcaption up for that. do I move figcaption tag to a separate line?

Comment (sils) [inline docs/overlay.html]: you even close the figcaption in an own line so it should have an own line for opening it, moving it down would increase readability IMO

**Comment by rhemon** [comment]
@sills brought `figcaption` tag to its own line

**Comment by sils** [comment]
Better, this is good IMO :)

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+language: generic
+services: docker
+script: docker run -v=$(pwd):/app --workdir=/app coala/base coala --ci -m MAJOR
```
Comment (sils) [inline .travis.yml]: hm you only look at MAJOR resuls? Too much spam in the normal ones? :/

Comment (rhemon) [inline .travis.yml]: yeah way too many

**Comment by rhemon** [comment]
@janschoenherr Sir, it would be nice if you could share your review on this PR. If there is any improvement you think is necessary, please let me know I will be working on it. It would be helpful if you shared you reason behind closing it. Thank you.

**Comment by janschoenherr** [comment]
UIKit version 2 has been moved to another branch: https://github.com/uikit/uikit/tree/v2/develop 

Please update the target branch of your pull request.

**Comment by jayvdb** [comment]
@janschoenherr, could you re-open the PR, so it can be amended?

**Comment by jayvdb** [comment]
Or would you prefer to keep this PR closed, for reference, and a new pull request opened?

**Pull request closed without merging**